config/opal_check_cuda.m4

```diff
@@ -143,6 +155,10 @@ AM_CONDITIONAL([OPAL_cuda_gdr_support], [test "x$CUDA_VERSION_60_OR_GREATER" = "
 AC_DEFINE_UNQUOTED([OPAL_CUDA_GDR_SUPPORT],$CUDA_VERSION_60_OR_GREATER,
                    [Whether we have CUDA GDR support available])
 
+# Set a macro to indicate that a workaround is needed for detecting GPU memory
+AC_DEFINE_UNQUOTED([OPAL_CUDA_GPU_MEM_WAR],$CUDA_VERSION_70_OR_LESS,
+                   [Whether we need the WAR for checking GPU memory])
+
```

Comment:
Is this `AC_COMPILE_IFELSE` and `AC_DEFINE_UNQUOTED` actually necessary?

You could just:

```c
static int mca_common_cuda_gpu_mem_check_workaround = (CUDA_VERSION > 7000) ? 0 : 1;
```